chaintesting/action_test_helpers.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package chaintesting
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/state"
+)
+
+var _ state.Mutable = (*inMemoryStore)(nil)
+
+// inMemoryStore is a storage that acts as a wrapper around a map and implements state.Mutable.
+type inMemoryStore struct {
+	Storage map[string][]byte
+}
+
+func NewInMemoryStore() *inMemoryStore {
+	return &inMemoryStore{
+		Storage: make(map[string][]byte),
+	}
+}
+
+func (s *inMemoryStore) GetValue(_ context.Context, key []byte) ([]byte, error) {
```

Comment:
nit: make the receiver `i` (to be in-line with the struct name)

Reply:
Replaced!